deepchecks/utils/plot.py

```diff
@@ -158,3 +169,51 @@ def shifted_color_map(cmap, start=0, midpoint=0.5, stop=1.0, name: str = 'shifte
 def hex_to_rgba(h, alpha):
     """Convert color value in hex format to rgba format with alpha transparency."""
     return 'rgba' + str(tuple([int(h.lstrip('#')[i:i+2], 16) for i in (0, 2, 4)] + [alpha]))
+
+
+def create_confusion_matrix_figure(confusion_matrix, x, y, is_normalize):
+    """Create a confusion matrix figure.
+
+    Parameters
+    ----------
+    confusion_matrix: np.ndarray
+        array containing the confusion matrix.
+    x: np.ndarray
+        array containing x axis data.
+    y: np.ndarray
+        array containing y axis data.
+    is_normalize: bool
+        if True will also show normalized values by the true values.
+
+    Returns
+    -------
+    go.Figure
+        confusion matrix figure
+
+    """
+    if is_normalize:
+        confusion_matrix_norm = confusion_matrix.astype('float') / \
+            confusion_matrix.sum(axis=1)[:, np.newaxis] * 100
+        confusion_matrix_norm = np.vectorize(_format_number_if_not_nan)(confusion_matrix_norm)
+        texttemplate = '%{z}%<br>(%{text})'
+        colorbar_title = '% of<br>True Values'
+        plot_title = 'Percent Out of True Values (Count)'
+    else:
+        confusion_matrix_norm = confusion_matrix
+        colorbar_title = None
+        texttemplate = '%{text}'
+        plot_title = 'Value Count'
+
+    fig = go.Figure(data=go.Heatmap(
+                x=x,
+                y=y,
+                z=confusion_matrix_norm,
+                text=confusion_matrix,
+                texttemplate=texttemplate))
+    fig.data[0].colorbar.title = colorbar_title
+    fig.update_layout(title=plot_title)
+    fig.update_layout(width=600, height=600)
```

Comment:
don't we want to remove all fixed sizes from plots?

Reply:
it looks bad without it